roles/installer/tasks/database_configuration.yml

```diff
@@ -51,3 +56,18 @@
     awx_postgres_database: "{{ pg_config['resources'][0]['data']['database'] | b64decode }}"
     awx_postgres_port: "{{ pg_config['resources'][0]['data']['port'] | b64decode }}"
     awx_postgres_host: "{{ pg_config['resources'][0]['data']['host'] | b64decode }}"
+  no_log: true
+
+- name: Check for old PostgreSQL configuration secret
+  k8s_info:
+    kind: Secret
+    namespace: '{{ meta.namespace }}'
+    name: '{{ tower_old_postgres_configuration_secret }}'
+  register: old_pg_config
+  when: tower_old_postgres_configuration_secret | length
+  no_log: true
```

Comment:
Does this task print out the contents of the secret?

Reply:
Yes, with `-v` it prints out the base64 encoded contents of the secret, which can be decoded easily.  

```
"password": "dG93ZXJwYXNz"
```

```
$ echo "dG93ZXJwYXNz" | base64 --decode
towerpass
```